crates/sui-framework/packages/deepbook/sources/clob_v2.move

```diff
@@ -986,9 +1057,9 @@ module deepbook::clob_v2 {
                 clock::timestamp_ms(clock),
                 coin::into_balance(quote_coin),
             );
-            join(
-                &mut base_coin,
-                coin::from_balance(base_balance_filled, ctx),
+            balance::join(
+                coin::balance_mut(&mut base_coin),
```

Comment:
Is there a `coin` API that lets you directly smash in a `balance` (and should we consider adding one if not)?

Reply:
We could consider it, yes.